extensions/amp-story/1.0/test/validator-amp-story-sidebar.html

```diff
@@ -0,0 +1,63 @@
+<!--
```

Comment:
Would be good to also have another sample HTML test case that checks the failure cases (e.g. specifying `side`, `toolbar`, or `toolbar-target`)